www/js/intro.js

```diff
@@ -89,16 +89,21 @@ angular.module('emission.intro', ['emission.splash.startprefs',
       $scope.userEmail = userEmail;
       CommHelper.registerUser(function(successResult) {
           const uuid = successResult.uuid;
-          return EnketoSurvey.getAllSurveyAnswers("manual/user_profile_survey"
+          return EnketoSurvey.getAllSurveyAnswers('manual/user_profile_survey'
```

Comment:
aren't you going to use the get/set profile?

Reply:
We only display the survey at the login screen if user does not have profile. If it is their second time then we skip the user profile survey and let them edit their answer at the settings page

Reply:
I see. Note that you don't really need this, because we already track if the intro is complete (`INTRO_DONE`) and only go through the onboarding if intro is not DONE.

The only exception is if the user manually logs out, in which case you presumably do want to show the profile again so that they can edit their profile information.